### ISSUE TITLE
Ability to add a Metric from another table in the same DB in the notebook

### ISSUE THREAD

**Issue by mazameli** [metabase/metabase#15915]
In the notebook, it should be possible to add a metric e.g. from the Reviews table if you began your GUI query from the Orders table. Those tables aren't related by foreign key, but they're from the same database. Note that you _can_ join a saved question from the same database, so why not a Metric?

I say, "in the notebook," because in the case where you have a breakout, you would need to pick a column to join on, just like current flow for joining a saved question to your current query. I.e., I'm assuming you'd need to add an "unconnected" Metric like this via a Join block.

Sadly, right now it also appears you can't join a saved question or a Metric if you don't have a breakout in the notebook, but it seems like this should also be possible in a Join block.

![image](https://user-images.githubusercontent.com/2223916/117041073-69e0c000-acbf-11eb-8280-33cc5b090042.png)

Related to https://github.com/metabase/metabase/issues/15914

**Comment by brunobergher** [comment]
@mngr handled with Metrics v2, right? Close this?

**Comment by mngr** [comment]
Nope, currently you can pick metrics only from those defined on the data source chosen
